### ISSUE TITLE
custom ytdl path

### ISSUE THREAD

**Issue by selfisekai** [mpv-player/mpv#7940]
Before requesting a new feature make sure it hasn't been requested yet.
https://github.com/mpv-player/mpv/labels/meta%3Afeature-request

### Expected behavior of the wanted feature
provide the path to ytdl to force using it instead of $(which youtube-dl), like:
```
mpv --ytdl-path=/bin/custom-ytdl https://www.youtube.com/watch?v=fwA_0N5NyEY
```

### Alternative behavior of the wanted feature
no idea

### Log file

Even if you think it's not necessary at first, it might help us later to find
possible issues. Make a log file made with -v -v or --log-file=output.txt, paste
it to https://0x0.st/ or attach it to the github issue, and replace this text
with a link to it.
nani?

**Comment by mia-0** [comment]
As a workaround, you can create a symlink in one of mpv’s config paths.

**Comment by CounterPillow** [comment]
Can't you just prepend `PATH` with a path to your ytdl binary? This is the usual application independent way to do something like this.

**Comment by selfisekai** [comment]
>Can't you just prepend PATH with a path to your ytdl binary?

@CounterPillow my point actually is to fork the ytdl and maintain it under a different name, without some kind of symlinks

**Comment by thebombzen** [comment]
I agree there should be an option `--ytdl-path` which points to a path to youtube-dl (or with another filename) so you don't have to pathmunge or symlink to make this work

**Comment by wiiaboo** [comment]
Implemented in #8207
